contracts/LightClientBridge.sol

```diff
@@ -116,25 +124,19 @@ contract LightClientBridge {
      * @param statement contains the statement signed by the validator(s)
      * @param randomSignatureCommitments an array of signatures from the randomly chosen validators
      * @param randomSignatureBitfieldPositions an array of bitfields from the chosen validators
```

Comment:
It would be much more efficient to just submit the signature bitfield instead. This consumes N bytes for N signers, while a bitfield consumes M bits for M validators. The bitfield is 8x more efficient in the best case.